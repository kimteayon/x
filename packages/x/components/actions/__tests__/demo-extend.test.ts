import { extendTest } from '../../../tests/shared/demoTest';

extendTest('actions');
