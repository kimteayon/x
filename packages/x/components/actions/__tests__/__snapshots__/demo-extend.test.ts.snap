// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders components/actions/demo/basic.tsx extend context correctly 1`] = `
<div
  class="ant-actions"
>
  <div
    class="ant-actions-list borderless"
  >
    <div
      class="ant-actions-list-item"
    >
      <div
        aria-describedby="test-id"
        class="ant-actions-list-item-icon"
      >
        <span
          aria-label="redo"
          class="anticon anticon-redo"
          role="img"
        >
          <svg
            aria-hidden="true"
            data-icon="redo"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M758.2 839.1C851.8 765.9 912 651.9 912 523.9 912 303 733.5 124.3 512.6 124 291.4 123.7 112 302.8 112 523.9c0 125.2 57.5 236.9 147.6 310.2 3.5 2.8 8.6 2.2 11.4-1.3l39.4-50.5c2.7-3.4 2.1-8.3-1.2-11.1-8.1-6.6-15.9-13.7-23.4-21.2a318.64 318.64 0 01-68.6-101.7C200.4 609 192 567.1 192 523.9s8.4-85.1 25.1-124.5c16.1-38.1 39.2-72.3 68.6-101.7 29.4-29.4 63.6-52.5 101.7-68.6C426.9 212.4 468.8 204 512 204s85.1 8.4 124.5 25.1c38.1 16.1 72.3 39.2 101.7 68.6 29.4 29.4 52.5 63.6 68.6 101.7 16.7 39.4 25.1 81.3 25.1 124.5s-8.4 85.1-25.1 124.5a318.64 318.64 0 01-68.6 101.7c-9.3 9.3-19.1 18-29.3 26L668.2 724a8 8 0 00-14.1 3l-39.6 162.2c-1.2 5 2.6 9.9 7.7 9.9l167 .8c6.7 0 10.5-7.7 6.3-12.9l-37.3-47.9z"
            />
          </svg>
        </span>
      </div>
      <div
        class="ant-tooltip ant-zoom-big-fast-appear ant-zoom-big-fast-appear-prepare ant-zoom-big-fast ant-tooltip-placement-top"
        style="--arrow-x: 0px; --arrow-y: 0px; left: -1000vw; top: -1000vh; right: auto; bottom: auto; box-sizing: border-box;"
      >
        <div
          class="ant-tooltip-arrow"
          style="position: absolute; bottom: 0px; left: 0px;"
        />
        <div
          class="ant-tooltip-content"
        >
          <div
            class="ant-tooltip-inner"
            id="test-id"
            role="tooltip"
          >
            Retry
          </div>
        </div>
      </div>
    </div>
    <div
      class="ant-actions-list-item"
    >
      <div
        aria-describedby="test-id"
        class="ant-actions-list-item-icon"
      >
        <span
          aria-label="copy"
          class="anticon anticon-copy"
          role="img"
        >
          <svg
            aria-hidden="true"
            data-icon="copy"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M832 64H296c-4.4 0-8 3.6-8 8v56c0 4.4 3.6 8 8 8h496v688c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8V96c0-17.7-14.3-32-32-32zM704 192H192c-17.7 0-32 14.3-32 32v530.7c0 8.5 3.4 16.6 9.4 22.6l173.3 173.3c2.2 2.2 4.7 4 7.4 5.5v1.9h4.2c3.5 1.3 7.2 2 11 2H704c17.7 0 32-14.3 32-32V224c0-17.7-14.3-32-32-32zM350 856.2L263.9 770H350v86.2zM664 888H414V746c0-22.1-17.9-40-40-40H232V264h432v624z"
            />
          </svg>
        </span>
      </div>
      <div
        class="ant-tooltip ant-zoom-big-fast-appear ant-zoom-big-fast-appear-prepare ant-zoom-big-fast ant-tooltip-placement-top"
        style="--arrow-x: 0px; --arrow-y: 0px; left: -1000vw; top: -1000vh; right: auto; bottom: auto; box-sizing: border-box;"
      >
        <div
          class="ant-tooltip-arrow"
          style="position: absolute; bottom: 0px; left: 0px;"
        />
        <div
          class="ant-tooltip-content"
        >
          <div
            class="ant-tooltip-inner"
            id="test-id"
            role="tooltip"
          >
            Copy
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`renders components/actions/demo/basic.tsx extend context correctly 2`] = `[]`;

exports[`renders components/actions/demo/sub.tsx extend context correctly 1`] = `
<div
  class="ant-actions"
>
  <div
    class="ant-actions-list borderless"
  >
    <div
      class="ant-actions-list-item"
    >
      <div
        aria-describedby="test-id"
        class="ant-actions-list-item-icon"
      >
        <span
          aria-label="redo"
          class="anticon anticon-redo"
          role="img"
        >
          <svg
            aria-hidden="true"
            data-icon="redo"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M758.2 839.1C851.8 765.9 912 651.9 912 523.9 912 303 733.5 124.3 512.6 124 291.4 123.7 112 302.8 112 523.9c0 125.2 57.5 236.9 147.6 310.2 3.5 2.8 8.6 2.2 11.4-1.3l39.4-50.5c2.7-3.4 2.1-8.3-1.2-11.1-8.1-6.6-15.9-13.7-23.4-21.2a318.64 318.64 0 01-68.6-101.7C200.4 609 192 567.1 192 523.9s8.4-85.1 25.1-124.5c16.1-38.1 39.2-72.3 68.6-101.7 29.4-29.4 63.6-52.5 101.7-68.6C426.9 212.4 468.8 204 512 204s85.1 8.4 124.5 25.1c38.1 16.1 72.3 39.2 101.7 68.6 29.4 29.4 52.5 63.6 68.6 101.7 16.7 39.4 25.1 81.3 25.1 124.5s-8.4 85.1-25.1 124.5a318.64 318.64 0 01-68.6 101.7c-9.3 9.3-19.1 18-29.3 26L668.2 724a8 8 0 00-14.1 3l-39.6 162.2c-1.2 5 2.6 9.9 7.7 9.9l167 .8c6.7 0 10.5-7.7 6.3-12.9l-37.3-47.9z"
            />
          </svg>
        </span>
      </div>
      <div
        class="ant-tooltip ant-zoom-big-fast-appear ant-zoom-big-fast-appear-prepare ant-zoom-big-fast ant-tooltip-placement-top"
        style="--arrow-x: 0px; --arrow-y: 0px; left: -1000vw; top: -1000vh; right: auto; bottom: auto; box-sizing: border-box;"
      >
        <div
          class="ant-tooltip-arrow"
          style="position: absolute; bottom: 0px; left: 0px;"
        />
        <div
          class="ant-tooltip-content"
        >
          <div
            class="ant-tooltip-inner"
            id="test-id"
            role="tooltip"
          >
            Retry
          </div>
        </div>
      </div>
    </div>
    <div
      class="ant-actions-list-item"
    >
      <div
        aria-describedby="test-id"
        class="ant-actions-list-item-icon"
      >
        <span
          aria-label="copy"
          class="anticon anticon-copy"
          role="img"
        >
          <svg
            aria-hidden="true"
            data-icon="copy"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M832 64H296c-4.4 0-8 3.6-8 8v56c0 4.4 3.6 8 8 8h496v688c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8V96c0-17.7-14.3-32-32-32zM704 192H192c-17.7 0-32 14.3-32 32v530.7c0 8.5 3.4 16.6 9.4 22.6l173.3 173.3c2.2 2.2 4.7 4 7.4 5.5v1.9h4.2c3.5 1.3 7.2 2 11 2H704c17.7 0 32-14.3 32-32V224c0-17.7-14.3-32-32-32zM350 856.2L263.9 770H350v86.2zM664 888H414V746c0-22.1-17.9-40-40-40H232V264h432v624z"
            />
          </svg>
        </span>
      </div>
      <div
        class="ant-tooltip ant-zoom-big-fast-appear ant-zoom-big-fast-appear-prepare ant-zoom-big-fast ant-tooltip-placement-top"
        style="--arrow-x: 0px; --arrow-y: 0px; left: -1000vw; top: -1000vh; right: auto; bottom: auto; box-sizing: border-box;"
      >
        <div
          class="ant-tooltip-arrow"
          style="position: absolute; bottom: 0px; left: 0px;"
        />
        <div
          class="ant-tooltip-content"
        >
          <div
            class="ant-tooltip-inner"
            id="test-id"
            role="tooltip"
          >
            Copy
          </div>
        </div>
      </div>
    </div>
    <div
      class="ant-dropdown-trigger ant-actions-list-item"
    >
      <div
        class="ant-actions-list-item-icon"
      >
        <span
          aria-label="ellipsis"
          class="anticon anticon-ellipsis"
          role="img"
        >
          <svg
            aria-hidden="true"
            data-icon="ellipsis"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M176 511a56 56 0 10112 0 56 56 0 10-112 0zm280 0a56 56 0 10112 0 56 56 0 10-112 0zm280 0a56 56 0 10112 0 56 56 0 10-112 0z"
            />
          </svg>
        </span>
      </div>
    </div>
    <div
      class="ant-dropdown ant-slide-up-appear ant-slide-up-appear-prepare ant-slide-up ant-actions-sub-item ant-dropdown-show-arrow ant-dropdown-placement-bottomLeft"
      style="--arrow-x: 0px; --arrow-y: 0px; left: -1000vw; top: -1000vh; right: auto; bottom: auto; box-sizing: border-box;"
    >
      <div
        class="ant-dropdown-arrow"
      />
      <ul
        class="ant-dropdown-menu ant-dropdown-menu-root ant-dropdown-menu-vertical ant-dropdown-menu-light"
        data-menu-list="true"
        role="menu"
        tabindex="0"
      >
        <li
          class="ant-dropdown-menu-submenu ant-dropdown-menu-submenu-vertical"
          role="none"
        >
          <div
            aria-controls="rc-menu-uuid-test-share-popup"
            aria-expanded="false"
            aria-haspopup="true"
            class="ant-dropdown-menu-submenu-title"
            data-menu-id="rc-menu-uuid-test-share"
            role="menuitem"
            tabindex="-1"
          >
            <span
              aria-label="share-alt"
              class="anticon anticon-share-alt ant-dropdown-menu-item-icon"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="share-alt"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M752 664c-28.5 0-54.8 10-75.4 26.7L469.4 540.8a160.68 160.68 0 000-57.6l207.2-149.9C697.2 350 723.5 360 752 360c66.2 0 120-53.8 120-120s-53.8-120-120-120-120 53.8-120 120c0 11.6 1.6 22.7 4.7 33.3L439.9 415.8C410.7 377.1 364.3 352 312 352c-88.4 0-160 71.6-160 160s71.6 160 160 160c52.3 0 98.7-25.1 127.9-63.8l196.8 142.5c-3.1 10.6-4.7 21.8-4.7 33.3 0 66.2 53.8 120 120 120s120-53.8 120-120-53.8-120-120-120zm0-476c28.7 0 52 23.3 52 52s-23.3 52-52 52-52-23.3-52-52 23.3-52 52-52zM312 600c-48.5 0-88-39.5-88-88s39.5-88 88-88 88 39.5 88 88-39.5 88-88 88zm440 236c-28.7 0-52-23.3-52-52s23.3-52 52-52 52 23.3 52 52-23.3 52-52 52z"
                />
              </svg>
            </span>
            <span
              class="ant-dropdown-menu-title-content"
            >
              Share
            </span>
            <span
              class="ant-dropdown-menu-submenu-expand-icon ant-dropdown-menu-submenu-arrow"
            >
              <span
                aria-label="right"
                class="anticon anticon-right ant-dropdown-menu-submenu-arrow-icon"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  data-icon="right"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M765.7 486.8L314.9 134.7A7.97 7.97 0 00302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 000-50.4z"
                  />
                </svg>
              </span>
            </span>
          </div>
          <div
            class="ant-dropdown-menu-submenu ant-zoom-big-appear ant-zoom-big-appear-prepare ant-zoom-big ant-dropdown-menu-submenu-popup ant-dropdown-menu ant-dropdown-menu-light ant-dropdown-menu-submenu-placement-rightTop"
            style="--arrow-x: 0px; --arrow-y: 0px; left: -1000vw; top: -1000vh; right: auto; bottom: auto; box-sizing: border-box;"
          >
            <ul
              class="ant-dropdown-menu ant-dropdown-menu-sub ant-dropdown-menu-vertical"
              data-menu-list="true"
              id="rc-menu-uuid-test-share-popup"
              role="menu"
            >
              <li
                aria-describedby="test-id"
                class="ant-dropdown-menu-item ant-dropdown-menu-item-only-child"
                data-menu-id="rc-menu-uuid-test-qq"
                role="menuitem"
                tabindex="-1"
              >
                <span
                  class="ant-dropdown-menu-title-content"
                >
                  QQ
                </span>
              </li>
              <div
                class="ant-tooltip ant-zoom-big-fast-appear ant-zoom-big-fast-appear-prepare ant-zoom-big-fast ant-dropdown-menu-inline-collapsed-tooltip ant-tooltip-placement-right"
                style="--arrow-x: 0px; --arrow-y: 0px; left: -1000vw; top: -1000vh; right: auto; bottom: auto; box-sizing: border-box;"
              >
                <div
                  class="ant-tooltip-arrow"
                  style="position: absolute; top: 0px; left: 0px;"
                />
                <div
                  class="ant-tooltip-content"
                >
                  <div
                    class="ant-tooltip-inner"
                    id="test-id"
                    role="tooltip"
                  />
                </div>
              </div>
              <li
                aria-describedby="test-id"
                class="ant-dropdown-menu-item ant-dropdown-menu-item-only-child"
                data-menu-id="rc-menu-uuid-test-wechat"
                role="menuitem"
                tabindex="-1"
              >
                <span
                  class="ant-dropdown-menu-title-content"
                >
                  WeChat
                </span>
              </li>
              <div
                class="ant-tooltip ant-zoom-big-fast-appear ant-zoom-big-fast-appear-prepare ant-zoom-big-fast ant-dropdown-menu-inline-collapsed-tooltip ant-tooltip-placement-right"
                style="--arrow-x: 0px; --arrow-y: 0px; left: -1000vw; top: -1000vh; right: auto; bottom: auto; box-sizing: border-box;"
              >
                <div
                  class="ant-tooltip-arrow"
                  style="position: absolute; top: 0px; left: 0px;"
                />
                <div
                  class="ant-tooltip-content"
                >
                  <div
                    class="ant-tooltip-inner"
                    id="test-id"
                    role="tooltip"
                  />
                </div>
              </div>
            </ul>
          </div>
        </li>
        <li
          aria-describedby="test-id"
          class="ant-dropdown-menu-item ant-dropdown-menu-item-only-child"
          data-menu-id="rc-menu-uuid-test-import"
          role="menuitem"
          tabindex="-1"
        >
          <span
            class="ant-dropdown-menu-title-content"
          >
            Import
          </span>
        </li>
        <div
          class="ant-tooltip ant-zoom-big-fast-appear ant-zoom-big-fast-appear-prepare ant-zoom-big-fast ant-dropdown-menu-inline-collapsed-tooltip ant-tooltip-placement-right"
          style="--arrow-x: 0px; --arrow-y: 0px; left: -1000vw; top: -1000vh; right: auto; bottom: auto; box-sizing: border-box;"
        >
          <div
            class="ant-tooltip-arrow"
            style="position: absolute; top: 0px; left: 0px;"
          />
          <div
            class="ant-tooltip-content"
          >
            <div
              class="ant-tooltip-inner"
              id="test-id"
              role="tooltip"
            />
          </div>
        </div>
        <li
          aria-describedby="test-id"
          class="ant-dropdown-menu-item ant-dropdown-menu-item-danger"
          data-menu-id="rc-menu-uuid-test-delete"
          role="menuitem"
          tabindex="-1"
        >
          <span
            aria-label="delete"
            class="anticon anticon-delete ant-dropdown-menu-item-icon"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="delete"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M360 184h-8c4.4 0 8-3.6 8-8v8h304v-8c0 4.4 3.6 8 8 8h-8v72h72v-80c0-35.3-28.7-64-64-64H352c-35.3 0-64 28.7-64 64v80h72v-72zm504 72H160c-17.7 0-32 14.3-32 32v32c0 4.4 3.6 8 8 8h60.4l24.7 523c1.6 34.1 29.8 61 63.9 61h454c34.2 0 62.3-26.8 63.9-61l24.7-523H888c4.4 0 8-3.6 8-8v-32c0-17.7-14.3-32-32-32zM731.3 840H292.7l-24.2-512h487l-24.2 512z"
              />
            </svg>
          </span>
          <span
            class="ant-dropdown-menu-title-content"
          >
            Delete
          </span>
        </li>
        <div
          class="ant-tooltip ant-zoom-big-fast-appear ant-zoom-big-fast-appear-prepare ant-zoom-big-fast ant-dropdown-menu-inline-collapsed-tooltip ant-tooltip-placement-right"
          style="--arrow-x: 0px; --arrow-y: 0px; left: -1000vw; top: -1000vh; right: auto; bottom: auto; box-sizing: border-box;"
        >
          <div
            class="ant-tooltip-arrow"
            style="position: absolute; top: 0px; left: 0px;"
          />
          <div
            class="ant-tooltip-content"
          >
            <div
              class="ant-tooltip-inner"
              id="test-id"
              role="tooltip"
            />
          </div>
        </div>
      </ul>
      <div
        aria-hidden="true"
        style="display: none;"
      />
    </div>
    <div
      class="ant-actions-list-item"
    >
      <div
        aria-describedby="test-id"
        class="ant-actions-list-item-icon"
      >
        <span
          aria-label="delete"
          class="anticon anticon-delete"
          role="img"
        >
          <svg
            aria-hidden="true"
            data-icon="delete"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M360 184h-8c4.4 0 8-3.6 8-8v8h304v-8c0 4.4 3.6 8 8 8h-8v72h72v-80c0-35.3-28.7-64-64-64H352c-35.3 0-64 28.7-64 64v80h72v-72zm504 72H160c-17.7 0-32 14.3-32 32v32c0 4.4 3.6 8 8 8h60.4l24.7 523c1.6 34.1 29.8 61 63.9 61h454c34.2 0 62.3-26.8 63.9-61l24.7-523H888c4.4 0 8-3.6 8-8v-32c0-17.7-14.3-32-32-32zM731.3 840H292.7l-24.2-512h487l-24.2 512z"
            />
          </svg>
        </span>
      </div>
      <div
        class="ant-tooltip ant-zoom-big-fast-appear ant-zoom-big-fast-appear-prepare ant-zoom-big-fast ant-tooltip-placement-top"
        style="--arrow-x: 0px; --arrow-y: 0px; left: -1000vw; top: -1000vh; right: auto; bottom: auto; box-sizing: border-box;"
      >
        <div
          class="ant-tooltip-arrow"
          style="position: absolute; bottom: 0px; left: 0px;"
        />
        <div
          class="ant-tooltip-content"
        >
          <div
            class="ant-tooltip-inner"
            id="test-id"
            role="tooltip"
          >
            Clear
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`renders components/actions/demo/sub.tsx extend context correctly 2`] = `
[
  "Unknown event handler property \`%s\`. It will be ignored.",
]
`;

exports[`renders components/actions/demo/variant.tsx extend context correctly 1`] = `
<div
  class="ant-actions"
>
  <div
    class="ant-actions-list border"
  >
    <div
      class="ant-actions-list-item"
    >
      <div
        aria-describedby="test-id"
        class="ant-actions-list-item-icon"
      >
        <span
          aria-label="redo"
          class="anticon anticon-redo"
          role="img"
        >
          <svg
            aria-hidden="true"
            data-icon="redo"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M758.2 839.1C851.8 765.9 912 651.9 912 523.9 912 303 733.5 124.3 512.6 124 291.4 123.7 112 302.8 112 523.9c0 125.2 57.5 236.9 147.6 310.2 3.5 2.8 8.6 2.2 11.4-1.3l39.4-50.5c2.7-3.4 2.1-8.3-1.2-11.1-8.1-6.6-15.9-13.7-23.4-21.2a318.64 318.64 0 01-68.6-101.7C200.4 609 192 567.1 192 523.9s8.4-85.1 25.1-124.5c16.1-38.1 39.2-72.3 68.6-101.7 29.4-29.4 63.6-52.5 101.7-68.6C426.9 212.4 468.8 204 512 204s85.1 8.4 124.5 25.1c38.1 16.1 72.3 39.2 101.7 68.6 29.4 29.4 52.5 63.6 68.6 101.7 16.7 39.4 25.1 81.3 25.1 124.5s-8.4 85.1-25.1 124.5a318.64 318.64 0 01-68.6 101.7c-9.3 9.3-19.1 18-29.3 26L668.2 724a8 8 0 00-14.1 3l-39.6 162.2c-1.2 5 2.6 9.9 7.7 9.9l167 .8c6.7 0 10.5-7.7 6.3-12.9l-37.3-47.9z"
            />
          </svg>
        </span>
      </div>
      <div
        class="ant-tooltip ant-zoom-big-fast-appear ant-zoom-big-fast-appear-prepare ant-zoom-big-fast ant-tooltip-placement-top"
        style="--arrow-x: 0px; --arrow-y: 0px; left: -1000vw; top: -1000vh; right: auto; bottom: auto; box-sizing: border-box;"
      >
        <div
          class="ant-tooltip-arrow"
          style="position: absolute; bottom: 0px; left: 0px;"
        />
        <div
          class="ant-tooltip-content"
        >
          <div
            class="ant-tooltip-inner"
            id="test-id"
            role="tooltip"
          >
            Retry
          </div>
        </div>
      </div>
    </div>
    <div
      class="ant-actions-list-item"
    >
      <div
        aria-describedby="test-id"
        class="ant-actions-list-item-icon"
      >
        <span
          aria-label="copy"
          class="anticon anticon-copy"
          role="img"
        >
          <svg
            aria-hidden="true"
            data-icon="copy"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M832 64H296c-4.4 0-8 3.6-8 8v56c0 4.4 3.6 8 8 8h496v688c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8V96c0-17.7-14.3-32-32-32zM704 192H192c-17.7 0-32 14.3-32 32v530.7c0 8.5 3.4 16.6 9.4 22.6l173.3 173.3c2.2 2.2 4.7 4 7.4 5.5v1.9h4.2c3.5 1.3 7.2 2 11 2H704c17.7 0 32-14.3 32-32V224c0-17.7-14.3-32-32-32zM350 856.2L263.9 770H350v86.2zM664 888H414V746c0-22.1-17.9-40-40-40H232V264h432v624z"
            />
          </svg>
        </span>
      </div>
      <div
        class="ant-tooltip ant-zoom-big-fast-appear ant-zoom-big-fast-appear-prepare ant-zoom-big-fast ant-tooltip-placement-top"
        style="--arrow-x: 0px; --arrow-y: 0px; left: -1000vw; top: -1000vh; right: auto; bottom: auto; box-sizing: border-box;"
      >
        <div
          class="ant-tooltip-arrow"
          style="position: absolute; bottom: 0px; left: 0px;"
        />
        <div
          class="ant-tooltip-content"
        >
          <div
            class="ant-tooltip-inner"
            id="test-id"
            role="tooltip"
          >
            Copy
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`renders components/actions/demo/variant.tsx extend context correctly 2`] = `[]`;
